cypress/e2e/po/edit/repositories.po.ts

```diff
@@ -0,0 +1,56 @@
+import PagePo from '@/cypress/e2e/po/pages/page.po';
+import LabeledInputPo from '@/cypress/e2e/po/components/labeled-input.po';
+import AsyncButtonPo from '@/cypress/e2e/po/components/async-button.po';
+import LabeledSelectPo from '@/cypress/e2e/po/components/labeled-select.po';
+import RadioGroupInputPo from '@/cypress/e2e/po/components/radio-group-input.po';
+
+export default class RepositoriesCreateEditPo extends PagePo {
```

Comment:
There's a couple of different `Repositories` concepts, would suggest this one is referred to as `ChartRepositories` (as it's specific to helm chart repos)